documentation/Configs.md

```diff
@@ -16,31 +16,29 @@ Note that for both the locally installed system and cloudbased flowmanager mode,
 Since running the system in local install mode is expected to configure the machine and launch applications on login, it is dependent on the platform. For this reason, running the system locally is done from the platform specific folders - that is windows, linux or android.
 
 If you have installed GPII in the folder <GPII-install>:
-* In windows, go to <GPII-install>\windows
-* In linux, go to <GPII-install>/linux
-* In android, go to <GPII-install>/android
+* In Microsoft Windows, go to <GPII-install>\windows
+* In GNU Linux, go to <GPII-install>/linux
+* In Android, go to <GPII-install>/android
 
 Then run the command: `grunt start`
 
 By default this will start up the system with everything running locally, using the `development.all.local.json` configuration file of universal repository (see the `universal/gpii/configs` folder). If you would like to run the system using a different configuration, say `dev.remote.prefs` run the following:
-* in Windows: `SET NODE_ENV=dev.remote.prefs`
-* in linux: `EXPORT NODE_ENV=dev.remote.prefs`
+* in Microsoft Windows: `SET NODE_ENV=dev.remote.prefs`
+* in GNU Linux: `EXPORT NODE_ENV=dev.remote.prefs`
 
 Followed by the `grunt start` command.
 
 
-#### Cloud Based Flowmanager
+#### Cloud-Based Flowmanager
 
-Since the Cloud Based Flowmanager does not require any platform specific bindings (it only returns a modified set of settings - doesn't configure anything), you can run it from universal. Given that you have installed GPII in the folder <GPII-install>, go to: `<GPII-install>/node_modules/universal`.
+Since the Cloud-Based Flowmanager does not require any platform-specific bindings (it only returns a modified set of settings - but does not configure anything), you can run it from universal. Given that you have installed GPII in the folder <GPII-install>, go to: `<GPII-install>/node_modules/universal`.
 
 Change the NODE_ENV environment variable to the cloudbased config file:
-* in Windows: `SET NODE_ENV=cloudBased.development.all.local`
-* in linux: `EXPORT NODE_ENV=cloudBased.development.all.local`
+* in Microsoft Windows: `SET NODE_ENV=cloudBased.development.all.local`
+* in GNU Linux: `EXPORT NODE_ENV=cloudBased.development.all.local`
```

Comment:
Well, it's a good point, but since in this doc we're talking about the configuration system, I don't think it's the right place to mention these. My point is that the "export" command is, AFAIK, in every UNIX-like OS, such as GNU/Linux, FreeBSD, etc